korman/properties/modifiers/gui.py

```diff
@@ -0,0 +1,254 @@
+    pl_id = "journalbookmod"
+
+    bl_category = "GUI"
+    bl_label = "Journal"
+    bl_description = "Journal Book"
+    bl_icon = "WORDWRAP_ON"
+
+    versions = EnumProperty(name="Export Targets",
+                            description="Plasma versions for which this journal exports",
+                            items=game_versions,
+                            options={"ENUM_FLAG"},
+                            default=set(["pvMoul",]))#set(list(zip(*game_versions))[0]))
+    start_state = EnumProperty(name="Start",
+                               description="State of journal when activated",
+                               items=[("OPEN", "Open", "Journal will start opened to first page"),
+                                      ("CLOSED", "Closed", "Journal will start closed showing cover")],
+                               default="CLOSED")
+    book_type = EnumProperty(name="Book Type",
+                             description="GUI type to be used for the journal",
+                             items=[("bkBook", "Book", "A journal written on worn, yellowed paper"),
+                                    ("bkNotebook", "Notebook", "A journal written on white, lined paper")],
+                             default="bkBook")
+    book_scale_w = IntProperty(name="Book Width Scale",
+                               description="Width scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    book_scale_h = IntProperty(name="Book Height Scale",
+                               description="Height scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    book_source_locpath = StringProperty(name="Book Source LocPath",
+                                         description="LocPath for book's text (MO:UL)",
+                                         default="Global.Journals.Empty")
+    book_source_filename = StringProperty(name="Book Source Filename",
+                                          description="Filename for book's text (Uru:CC)",
+                                          default="")
+    book_source_name = StringProperty(name="Book Source Name",
+                                      description="Name of xJournalBookDefs.py entry for book's text (Uru:ABM)",
+                                      default="Dummy")
+    clickable_region = PointerProperty(name="Region",
+                                       description="Region inside which the avatar must stand to be able to open the journal (optional)",
+                                       type=bpy.types.Object,
+                                       poll=idprops.poll_mesh_objects)
+
+    def export(self, exporter, bo, so):
+        our_versions = [globals()[j] for j in self.versions]
+        version = exporter.mgr.getVer()
+        if version not in our_versions:
+            return
```

Comment:
Might want to print a warning to the log about this?

Reply:
I would assume that the artist isn't exporting to a version they haven't selected, so a warning might be superfluous... *unless* the mod has no versions selected at all.  That would be much more likely to indicate an unintentional situation.  What do you think?

Reply:
I'm thinking after some time if an artist exports to a version not supported by their modifier, they might be confused when the journal doesn't work. Because time keeps on slippin' slipping' 😄

Reply:
That's a good point.  Having it show up in the log would make that much quicker to debug.